agora.ctmpl

```diff
@@ -0,0 +1,116 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{$keyname := printf "secret/dsde/%s/agora/agora.conf" $environment}}
+{{with vault $keyname}}
+
+webservice {
+    port = "{{.Data.webservice_port}}"
+    interface = "0.0.0.0"
+    host = "{{.Data.webservice_host}}"
+    schema = "https"
+}
+
+instance {
+    name = "reference"
+}
+
+mongodb {
+    host = "{{.Data.mongodb_host}}"
+    port = 27017
+    user = "{{.Data.mongodb_user}}"
+    password = "{{.Data.mongodb_password}}"
+    db = "{{.Data.mongodb_db}}"
+}
+
+environment = "dev"
```

Comment:
What is this key, and why is it hardcoded "dev"? How does it relate to the environment set at line1?